src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptGoToSymbolContext.cs

```diff
@@ -23,7 +23,7 @@ internal VSTypeScriptGoToSymbolContext(Document document, int position, Cancella
         public int Position { get; }
         public CancellationToken CancellationToken { get; }
 
-        public TextSpan Span { get; }
+        public TextSpan Span { get; set; }
```

Comment:
It was a msitake removing this setter.  The expectation is that TS *will* set this property, and we will then take the value and use it to determine what to underline.